api/src/main/java/org/pmiops/workbench/actionaudit/ActionAuditQueryServiceImpl.java

```diff
@@ -39,8 +39,7 @@
           + "WHERE %s\n"
           + "  AND @after <= TIMESTAMP_MILLIS(CAST(jsonPayload.timestamp AS INT64))\n"
           + "  AND TIMESTAMP_MILLIS(CAST(jsonPayload.timestamp AS INT64)) < @before\n"
-          + "  AND @after_partition_time <= _PARTITIONTIME\n"
-          + "  AND _PARTITIONTIME < @before_partition_time\n"
+          + "  AND @after_partition_time <= %s AND %<s < @before_partition_time\n"
```

Comment:
TIL `%<s` means repeat the previous instance of `%s`

Reply:
Good to know!